connectors/v2/22tracks.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+/* global Connector */
+
+Connector.playerSelector = '.player';
+
+Connector.artistTrackSelector = '.player__progress__title';
+
+Connector.isPlaying = function () {
+	return $('.icon-pause').length;
```

Comment:
Shouldn't this be more of a boolean expression? e.g. `$('.player).has('.icon-pause')`